cocotb/__init__.py

```diff
@@ -240,6 +240,23 @@ def _initialise_testbench(argv_):
 
     dut = cocotb.handle.SimHandle(handle)
 
+    try:
+        import pytest
+    except ImportError:
+        log.warning("Pytest not found, assertion rewriting will not occur")
```

Comment:
Maybe we could suggest ``pip install pytest`` here.

Reply:
Well... are we going for "optional", or "optional, but recommended"? I think we may have a runtime dependency on `pytest` in 2.0, so this is probably a good thing.

Reply:
Recommending `pip` in an error message is likely to get users into a trap who are using `conda`. The users who don't know how to fix it from this message alone are also the ones who are most likely to screw up when installing it.

Reply:
Should we be warning at all if it isn't loaded?